easybuild/easyconfigs/g/GPAW/GPAW-1.5.2-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '1.5.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+ method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or
+ atom-centered basis-functions."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('GPAW-1.4.0-customize.patch', 1),
+    'GPAW-1.5.2-slurm-submit-fix.patch',
+]
+checksums = [
+    'd2d6b4c1b9a61532e1e2f6200c4ca48835ec8275d182e6913088b3aa53773a27',  # gpaw-1.5.2.tar.gz
+    '0fe732e9b6bf793aebd6e248bd14a76fa0603dbdfde9f9e268cc7bf71b37b3a0',  # GPAW-1.4.0-customize.patch
+    '099924025c1ea4af2dcbad7bd259a8a4baf22c29236f115f47018cb597051c35',  # GPAW-1.5.2-slurm-submit-fix.patch
+]
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('ASE', '3.17.0', versionsuffix),
+    ('libxc', '3.0.1'),
+    ('libvdwxc', '0.3.2'),
+    ('GPAW-setups', '0.9.20000', '', True),
+]
+
+buildcmd = 'build --customize=eb_customize.py'
+install_target = 'install --customize=eb_customize.py'
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
this PYTHONPATH is not needed since PythonPackage adds it by default.
It also needs
download_dep_fail = True
use_pip = False

The same for the other one.

Reply:
Done!  Thanks for the fixes.